Jamescjay/Nobility-hub-client
# f Resolve Missing Module Error for `react-chat-engine-pretty`

This PR addresses the issue where the application was unable to locate the `react-chat-engine-pretty` module, resulting in a runtime error. The proposed changes include verifying the module name, installing an alternative if necessary, or refactoring the code to remove dependencies on this module.

### Changes Made
- **Removed `react-chat-engine-pretty`** from `package.json` as it is not available in the npm registry.
- **Installed `react-chat-engine`** as an alternative for chat functionality.
- Updated code references to use `react-chat-engine` where appropriate.

### How to Test
1. Pull the latest changes from this branch.
2. Run `npm install` to update dependencies.
3. Start the application with `npm start`.
4. Confirm that the application loads without missing module errors and that the chat functionality operates as expected.

### Linked Issue
Closes #40  – Fix `Cannot find module 'react-chat-engine-pretty'` Error
